### DISCUSSION TITLE
Add support on pre-commit manager for other datasources

### DISCUSSION THREAD

**Discussion by rimoley** [renovatebot/renovate#28494] [Request Help]
### Tell us more.

Hi,
currently pre-commit manager is only compatible with `github-tags or gitlab-tags datasources.
Due to availability, we have mirrored precommit hooks repositories in an internal bitbucket, so the proposal would be to allow other datasources like git-tags or bitbucket-tags to have more flexibility on where to looks form tags.

Our workaround for now, is to use a custom regex that tracks the version via github-tags, but this would required to modify all the precommit files in our repos, which it is doable yet inconvenient :)

Thanks :) 
Regards

**Comment by benjamb** [comment]
I'd also like to be able to force `pre-commit` to use a `git-tags` datasource, given the `github-tags` datasource doesn't work without an API token, which I would prefer to avoid creating.
